src/test/java/apicurito/tests/utils/slenide/CommonUtils.java

```diff
@@ -314,4 +318,8 @@ public static SelenideElement getElementRow(SelenideElement section, String sect
         //element not found
         return null;
     }
+
+    public static SelenideElement getLinkByText(String text) {
```

Comment:
unnecessary method? Would not be better to use $(By.linkText(text)); directly in method?